blade-graphics/src/gles/pipeline.rs

```diff
@@ -1,5 +1,6 @@
 use glow::HasContext as _;
 use naga::back::glsl;
+use crate::ColorTargetState;
```

Comment:
let's not change the style in which this module (and the library is written) - just do `crate::ColorTargetState` where it needs to be, no need to declare it here